src/provider/controller/NotificationCenter.ts

```diff
@@ -67,7 +67,7 @@ export class NotificationCenter extends AsyncInit {
         await this.hideWindowOffscreen();
         this._trayIcon.setIcon('https://openfin.co/favicon-32x32.png');
         this._trayIcon.onLeftClick.add(() => {
-            this._store.dispatch(new ToggleVisibility());
+            this._store.dispatch(new ToggleCenterVisibility(ToggleCenterVisibilitySource.TRAY));
         });
         await this.sizeToFit();
         await this.addListeners();
```

Comment:
Add a `this.toggleWindow(this._store.state.centerVisible)` so the center is initialized with initial state visibility from the store.

Reply:
Done slightly differently, but doing the same thing

✅